packages/ggsmark/index.js

```diff
@@ -86,7 +86,10 @@ export default (text) => {
         height: 378,
         disabled: false,
         replace: [
-          ['twitch.tv/videos/', 'player.twitch.tv/?parent=ggs.sx&video='],
+          [
+            'www.twitch.tv/videos/',
+            'www.player.twitch.tv/?parent=ggs.sx&video='
```

Comment:
```suggestion
            `www.player.twitch.tv/?${parents}&video=`
```
Should this be a variable instead? Hard-coding it solves the immediate problem, but what if we need to change it?

Reply:
We can't just add `parents` variables and not use it 😅

The idea was to expose options on the `ggsmark( '!(some twitch link)', options = {})`.

```js
ggsmark('!(some twitch link', {parents: ['ggs.sx', 'example.com', 'blah.blah'])
```

So then the string looks like this:

```js
`www.player.twitch.tv/?parent=ggs.sx&parent=example.com&parent=blah.blah&video=`
```

Reply:
If this is too difficult, I'll add it in

Reply:
I'll take a look in a couple of hours 😀